modules/database/src/admin/customEndpoints/utils.ts

```diff
@@ -131,6 +131,7 @@ function _queryValidation(
  * }
  */
 function _inputValidation(
+  operation: number,
```

Comment:
While you're at it, update `operation` and `location` to use enums instead of plain numbers.
We already have an `OperationsEnum` object type that should be turned into an actual enum.

Reply:
I think that should be fixed in another PR